source/sync/configure/enable-sync.txt

```diff
@@ -179,33 +179,27 @@ Procedure
                  -X GET \
                  -h 'Authorization: Bearer <Valid Access Token>'
 
-            Once you have the configuration, add the ``flexible_sync`` object with the
-            following template configuration:
+            For details on the sync configuration, refer to the :ref:`sync-configuration-reference`.
 
-            .. code-block:: json
+         .. step:: Enable Development Mode (optional)
 
-               {
-                 ...
-                 "flexible_sync": {
-                   "state": "enabled",
-                   "database_name": "<Name of Database>",
-                   "client_max_offline_days": <Number>,
-                   "is_recovery_mode_disabled": <Boolean>,
-                   "indexed_queryable_fields_names": <Array of String Field Names>,
-                   "queryable_fields_names": <Array of String Field Names>,
-                   "collection_queryable_fields_names": <Map[String][]String>
-                 }
-                 ...
-               }
+            You may need to enable :ref:`Development Mode <development-mode>` before deploying your changes. 
```

Comment:
why might I need to enable Dev Mode? Also, why would I set the config file _after_ getting it? wouldn't I want to `GET` the latest state before `PATCH`ing?

Reply:
In the prerequisites section it says that "Enabling Development Mode is helpful if the schemas aren't yet compatible with sync". On the development mode description, says it is helpful to "streamline production". I can do a reword to reflect this :)
As for your 2nd question, that is exactly what is dictated in the procedure; the user gets the config file first and then does a patch update. I think seeing the config file first is helpful for context. I can include a note to confirm the patch by getting the configuration again. Does that make sense?